pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -376,6 +376,7 @@ stdenv.mkDerivation ((drvAttrs config stdenv.hostPlatform.linux-kernel kernelPat
       ++ optionals (lib.versionAtLeast version "4.16") [ bison flex ]
       ++ optionals (lib.versionAtLeast version "5.2")  [ cpio pahole zlib ]
       ++ optional  (lib.versionAtLeast version "5.8")  elfutils
+      ++ optional  (lib.versionAtLeast version "6.6")  buildPackages.kmod
```

Comment:
Does this need to be `buildPackages`? I'm not sure if anything about kmod is target-specific but this might get weird for cross?

Reply:
Based on https://github.com/NixOS/nixpkgs/commit/50e2bb12f6385f4244a6c52bd9ba75015ead392e it does appear to have something to do with cross-compilation, but I don't know if it is still required.

Reply:
It shouldn't be required now because it's added to `nativeBuildInputs` which should be spliced.

Reply:
This should be an easy question to answer, right? We can try cross-compiling the kernel using just "kmod" and see if it works.  (I've started a build — will get back with the results.)

Reply:
Seems to work fine without the `buildPackages.` — no warning.